RealmSwift-swift2.2/Aliases.swift

```diff
@@ -42,12 +42,16 @@ import Realm
  * `Data`
  * `Date`
 
- ### Relationships: List (`Array`) and `Object` types
+ ### Relationships: `List` (array) and `Object` types
 
  * `Object`
- * `Array`
+ * `List`
```

Comment:
Since this is a type alias, it'll still be `PropertyType.Array` since the Objective-C version is `RLMPropertyTypeArray`